### DISCUSSION TITLE
[Feature] Sorting options in album

### DISCUSSION THREAD

**Discussion by vultore13** [immich-app/immich#1948] [Feature Request]
### The feature

Hi,

is it possible to get sorting options for specific albums, like photos from newest to oldest inside of album, but be able to set this per album?
Like album of kids, where you want to see newest on top and scroll back to old ones.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1689

**Reply by vultore13** [reply]
Sorry if i didn't say it clear, i want to be able to sort photos inside of album, not list of albums.

**Reply by bo0tzz** [reply]
Yes, that's also covered there
